app/src/main/java/pl/szczodrzynski/edziennik/ui/home/cards/HomeTimetableCard.kt

```diff
@@ -343,6 +343,19 @@ class HomeTimetableCard(
         b.nextLessons.text = text.concat("\n")
     }}
 
+    private fun adjustTimeWidth(time: String?): String {
+        if (time != null) {
+            //compared to 4 because when hour is less than 10 there are 4 chars (hour, colon and minutes)
+            if(time.length == 4) {
+                return " $time  "
+            } else {
+                return "$time "
+            }
+        } else {
+            return ""
+        }
+    }
```

Comment:
This can be replaced with a `when` block and the function converted to an expression body, like this:
```suggestion
    private fun adjustTimeWidth(time: String?) = when {
        time == null -> ""
        // compared to 4 because when hour is less than 10 there are 4 chars (hour, colon and minutes)
        time.length == 4 -> " $time "
        else -> "$time "
    }
```

Please check if this works, because I can't do that right now.